jobs/build/rhcos_sync/S3-rhcossync.sh

```diff
@@ -187,8 +187,8 @@ fi
 aws s3 sync --no-progress --delete --exact-timestamps ./ "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/${VERSION}/"
 aws s3 sync --no-progress --delete --exact-timestamps ./ "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/${VERSION}/" --profile cloudflare --endpoint-url ${CLOUDFLARE_ENDPOINT}
 if [ $NOLATEST -eq 0 ]; then
-    aws s3 sync --no-progress --delete --exact-timestamps "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/${VERSION}/" "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/latest/"
-    aws s3 sync --no-progress --delete --exact-timestamps "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/${VERSION}/" "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/latest/" --profile cloudflare --endpoint-url ${CLOUDFLARE_ENDPOINT}
+    aws s3 sync --no-progress --delete --exact-timestamps ./ "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/latest/"
+    aws s3 sync --no-progress --delete --exact-timestamps ./ "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/latest/" --profile cloudflare --endpoint-url ${CLOUDFLARE_ENDPOINT}
     emulateSymlinks "s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/${VERSION}/"
```

Comment:
So we are passing `"s3://art-srv-enterprise${BASEDIR}/${RHCOS_MIRROR_PREFIX}/${VERSION}/"` as path right?

Reply:
Also if we are using the local source, `S3_SOURCE="$1"` will be unused, so we can remove that as well

Reply:
yes, I was thinking about pass in local path `./` but then find in the emulateSymlinks function it will extract the version from the path

Reply:
updated in https://github.com/openshift-eng/aos-cd-jobs/pull/4253/commits/c690ab70ab5eb3112ac3a122b74da22a4c94a6b7